lib/appsignal/config.rb

```diff
@@ -39,7 +39,6 @@ class Config
       :enable_statsd                  => true,
       :ca_file_path                   => File.expand_path(File.join("../../../resources/cacert.pem"), __FILE__),
       :dns_servers                    => [],
-      :filter_data_keys               => [],
       :files_world_accessible         => true,
       :transaction_debug_mode         => false
     }.freeze
```

Comment:
It would be great to have `DEFAULT_CONFIG` keys ordered alphabetically.

Reply:
Yes, something for another PR :) I didn't want to do it here, because I was already making enough changes in 6 different repos.